packages/calcite-components/src/components/checkbox/checkbox.scss

```diff
@@ -4,16 +4,31 @@
  * These properties can be overridden using the component's tag as selector.
  *
  * @prop --calcite-checkbox-size: Specifies the component's height and width.
+ * @prop --calcite-checkbox-border-color: Specifies the component's color.
+ * @prop --calcite-checkbox-border-color-hover: Specifies the component's color when hovered.
+ * @prop --calcite-checkbox-color: Specifies the component's font color.
```

Comment:
Can we revisit this description? It is used to set the `color` prop, but from a user's perspective, it's setting the check's color. 

@alisonailea Do we need to specify `icon` in the name or is this an exception because the component is mostly an icon?

Reply:
we could use -checkbox-icon-color but I think -checkbox-color makes sense here but I'm open to either

Reply:
I'll leave it as `-checkbox-color`.

Reply:
Let’s use `-checkbox-icon-color` for consistency. Since we already differentiate the border color, I think sticking with the icon color makes sense.

Reply:
I updated the token to `--calcite-checkbox-icon-color` 🚀